components/automate-ui/src/app/pages/data-feed/data-feed.component.spec.ts

```diff
@@ -99,8 +99,8 @@ describe('DatafeedComponent', () => {
     });
 
     it('shows expected item count', () => {
-      const tableBody = getElementByCss(`${listId}`);
-      expect(tableBody.children.length).toBe(destinations.length + 1);
+      const tableBody = getElementByCss(`${listId} chef-table-body`);
+      expect(tableBody.children.length).toBe(destinations.length);
```

Comment:
Can you explain this discrepancy? (Why would the test result be different?)

Reply:
In the previous HTML markup, the table markup was semantically off - it was missing a `table-head` wrapping, so the list items inside the table included the header row - which required the test to need one extra list item, even though that item was actually a header row.

With the new HTML markup, the table now has a `chef-table-head` wrapping the header row.  A `chef-table-body` is holding the _actual_ table body contents.  So the number of list items should match only the number of items in the list.